### PR TITLE
Define PackageId for Queue implementations.

### PR DESCRIPTION
**Pull request by claudiamurialdo**
Issue:97389

### CODE REVIEW

**Review by ggallotti — DISMISSED** [review]

File: dotnet/src/dotnetcore/Providers/Messaging/GXAzureQueue/GXAzureQueue.csproj [hunk 1]
```diff
@@ -1,7 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
+    <PackageId>GeneXus.Azure.Queue</PackageId>
+    <PackageTags>Message Azure</PackageTags>
```
Comment (ggallotti) [inline dotnet/src/dotnetcore/Providers/Messaging/GXAzureQueue/GXAzureQueue.csproj:6]: Maybe "Azure Queue Messaging" for better discoverability?

File: dotnet/src/dotnetcore/Providers/Messaging/GXAmazonSQS/GXAmazonSQS.csproj [hunk 2]
```diff
@@ -1,7 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
+    <PackageId>GeneXus.AWS.Queue</PackageId>
+    <PackageTags>Message Amazon</PackageTags>
```
Comment (ggallotti) [inline dotnet/src/dotnetcore/Providers/Messaging/GXAmazonSQS/GXAmazonSQS.csproj:6]: Maybe "AWS SQS Queue Messaging" for better discoverability?

Comment (claudiamurialdo) [reply dotnet/src/dotnetcore/Providers/Messaging/GXAmazonSQS/GXAmazonSQS.csproj]: Done.

File: dotnet/src/dotnetcore/Providers/Messaging/GXAzureQueue/GXAzureQueue.csproj (hunk 1 above)
Comment (claudiamurialdo) [reply dotnet/src/dotnetcore/Providers/Messaging/GXAzureQueue/GXAzureQueue.csproj]: Done.

**Pull request merged**